docs/_includes/common/flags.html

```diff
@@ -1,8 +1,29 @@
 <div id="flags">
     <h2>Flags</h2>
     <p>Skin has the ability to switch between design system versions.</p>
-    <p>DS6 is the current design system. To opt into DS4, Lasso.js requires the <span class="highlight">ds-4</span> <a href="https://github.com/lasso-js/lasso#conditional-dependencies">conditional dependency flag</a>.</p>
+    <p>DS6 is the current design system but you can access older design system using flags. For example, you can access the older ds4 design system using  <span class="highlight">ds-4</span> flag.</p>
 
+    <h4>Webpack SetUp</h4>
+    <p>If you need to pull in the <span class="highlight">flags</span>, we should install and configure <span class="highlight">arc-webpack</span></p>
+
+    {% highlight js %}
+yarn add -D arc-webpack
+    {% endhighlight %}
+
+    <p>Do the following in your webpack config:</p>
+    {% highlight js %}
+const AdaptivePlugin = require("arc-webpack");
+...
+...
+    plugins: [
+        new AdaptivePlugin({ flags: { "ds-4": true } })
+    ]
+    {% endhighlight %}
+
+    <p>Note: Currently, there is a bug in <span class="highlight">arc-webpack</span> which is being fixed. The workaround is to enable <span class="highlight">MiniCssExtractPlugin</span> even in all environments</p>
```

Comment:
I've published the new version of `arc-webpack` that addresses this issue.  @raveenakothapally would you be able to update and confirm that this fixes the issue?  If so, we should remove this note.

Reply:
Will pull the new version and check